src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -807,10 +818,7 @@ const WorkflowView = _.flow(
     const data = currentSnapRedacted ?
       _.map(k => ({ name: k, inputType: 'unknown' }), _.keys(modifiedConfig[key])) :
       modifiedInputsOutputs[key]
-    const filteredData = _.flow(
-      key === 'inputs' && !includeOptionalInputs ? _.reject('optional') : _.identity,
-      _.sortBy(['optional', ({ name }) => name.toLowerCase()])
-    )(data)
+    const filteredData = key === 'inputs' && !includeOptionalInputs ? _.reject('optional')(data) : _.identity(data)
```

Comment:
nit: Without the `flow`, there's no need to use the curried form of `_.reject` or explicitly call `_.identity`. So this could be:
```suggestion
    const filteredData = key === 'inputs' && !includeOptionalInputs ? _.reject('optional', data) : data
```